Error when running marko-language-server.cmd on Windows

## @marko/language-server@1.3.6

<!--- Provide the package name and the exact version in which you see the bug.  You can run `npm ls <package-name>` to see this. -->

### Details
I've installed the marko language server, but when I run it from the terminal I receive an error

<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug.  How has this bug affected you? What were you trying to accomplish? -->

### Expected Behavior
Command to run without error
<!--- Tell us what should happen -->

### Actual Behavior
Error is thrown (see additional details)

<!--- Tell us what happens instead -->

### Possible Fix
It mentions not finding the `axe-core` module. Is this something I need to install myself?
<!--- Not obligatory, but suggest a fix or reason for the bug -->

<details><summary>Additional Info</summary>

### Your Environment

<!-- Include as many relevant details about the environment you experienced the bug in -->
- Windows 11
- Node.js v20.10.0

### Stack Trace
```
C:\src\ [main ≡]> marko-language-server.cmd   
node:internal/modules/cjs/loader:1147
  throw err;
  ^

Error: Cannot find module 'axe-core'
Require stack:
- C:\Users\<User>\AppData\Roaming\npm\node_modules\@marko\language-server\dist\index.js
- C:\Users\<User>\AppData\Roaming\npm\node_modules\@marko\language-server\bin.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (C:\Users\<User>\AppData\Roaming\npm\node_modules\@marko\language-server\dist\index.js:309:31)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Module.require (node:internal/modules/cjs/loader:1235:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\<User>\\AppData\\Roaming\\npm\\node_modules\\@marko\\language-server\\dist\\index.js',
    'C:\\Users\\<User>\\AppData\\Roaming\\npm\\node_modules\\@marko\\language-server\\bin.js'
  ]
}

Node.js v20.10.0
```
<!-- If an error is thrown, provide the stack trace here -->

</details>

--- a/.changeset/funny-steaks-run.md
+++ b/.changeset/funny-steaks-run.md
@@ -1,0 +1,5 @@
+---
+"@marko/language-server": patch
+---
+
+Fix dependency issue

--- a/package.json
+++ b/package.json
@@ -19,12 +19,7 @@
     "mocha-snap": "^5.0.0",
     "prettier": "^3.3.3",
     "tsx": "^4.19.2",
-    "typescript": "^5.6.3",
-    "@types/jsdom": "^21.1.7"
-  },
-  "dependencies": {
-    "axe-core": "^4.10.2",
-    "jsdom": "^25.0.1"
+    "typescript": "^5.6.3"
   },
   "private": true,
   "scripts": {

--- a/packages/language-server/package.json
+++ b/packages/language-server/package.json
@@ -21,10 +21,13 @@
     "vscode-css-languageservice": "^6.3.1",
     "vscode-languageserver": "^9.0.1",
     "vscode-languageserver-textdocument": "^1.0.12",
-    "vscode-uri": "^3.0.8"
+    "vscode-uri": "^3.0.8",
+    "axe-core": "^4.10.2",
+    "jsdom": "^25.0.1"
   },
   "devDependencies": {
     "@types/prettier": "^3.0.0",
+    "@types/jsdom": "21.1.7",
     "tsx": "^4.19.2"
   },
   "exports": {

